jobs/ci-kubernetes-e2e-gci-tip-gce.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o xtrace
+
+readonly testinfra="$(dirname "${0}")/.."
+
+### provider-env
+export KUBERNETES_PROVIDER="gce"
+export E2E_MIN_STARTUP_PODS="8"
+export KUBE_GCE_ZONE="us-central1-f"
+export FAIL_ON_GCP_RESOURCE_LEAK="true"
+export CLOUDSDK_CORE_PRINT_UNHANDLED_TRACEBACKS="1"
+
+### project-env
+# expected empty
+
+### job-env
+# This list should match the list in kubernetes-pull-build-test-e2e-gce.
+export GINKGO_TEST_ARGS="--ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]"
+export GINKGO_PARALLEL="y"
+export KUBE_OS_DISTRIBUTION="gci"
+
+# Use the latest gci-base prefixed version from container-vm-image-staging (this is the GCI canary)
+export KUBE_GCE_MASTER_PROJECT="container-vm-image-staging"
+export KUBE_GCE_NODE_PROJECT="container-vm-image-staging"
+export KUBE_GCI_VERSION=$(gcloud compute images list --project=container-vm-image-staging -r "gci-base.*" --sort-by=~creation_timestamp | head -n2 | tail -n1 | awk '{print $1}')
```

Comment:
If you just want to get the latest GCI build, you could use the `gci-canary` image family, as is done here: https://github.com/kubernetes/test-infra/blob/master/jobs/ci-kubernetes-e2e-gce-gci-ci-master.sh#L36